src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -174,7 +174,7 @@ def nwm_output_generator(
             d_df = flow_df.iloc[:,2::3]
 
         # replace waterbody lake_ids with outlet link ids
-        if link_lake_crosswalk:
+        if (link_lake_crosswalk and not link_lake_crosswalk=='NONE'):
```

Comment:
@JurgenZach-NOAA  `link_lake_crosswalk` shouldn't be "NONE", its a dictionary, it's either empty or have values in that. The fact that it become 'NONE" shows something is wrong. If you look at where this function being called in `__main__ .py` (line 1791) and put a trace before the function you'll see `link_lake_crosswalk` has a value, but right after the function being called it become "NONE". The reason is one input (`duplicate_ids_df`) is missing from the function when it being called